app/src/main/res/values/strings.xml

```diff
@@ -1322,6 +1322,7 @@
     <string name="talk_warn_submitted_and_saved">Message saved and published to editor\'s user talk page</string>
     <string name="talk_warn_saved_message">Select an existing warning message or compose a new message.</string>
     <string name="talk_warn_saved_message_dropdown">Saved messages</string>
+    <string name="talk_warn_saved_messages">Saved messages</string>
```

Comment:
Is this string identical to the one above it? Does it need to be separate?

Reply:
Yes, just because we do not have a `dropdown` anymore, i have deleted the old string [ in #4483] and created this new one.